elasticdl/python/tests/data_reader_test.py

```diff
@@ -59,9 +57,7 @@ def test_recordio_data_reader(self):
             # Test records reading
             records = list(
                 reader.read_records(
-                    _MockedTask(
-                        0, num_records, shard_name, elasticdl_pb2.TRAINING
-                    )
+                    _Task(shard_name, 0, num_records, elasticdl_pb2.TRAINING)
```

Comment:
Do we still need the `_MockedTask` class anymore?

Reply:
Yes, I have deleted it from line 43 to line 44.